requirements.txt

```
@@ -0,0 +1 @@
+protobuf==4.23.4
```

```suggestion
protobuf~=4.23.0
```

We're going to have at least minor updates, right?